README.md

```diff
@@ -1,38 +1,56 @@
 # Neural Network Compression Framework (NNCF)
 
-This repository contains a PyTorch\*-based framework and samples for neural networks compression.
+NNCF provides a suite of advanced compression algorithms, applied during a model fine-tuning and samples for models 
+compression. It is highly aligned with [OpenVINO&trade;](https://github.com/openvinotoolkit/openvino) and is aimed 
+to improve runtime performance of models in OpenVINO&trade; with minimal accuracy drop.
 
-The framework is organized as a Python\* package that can be built and used in a standalone mode.
-The framework architecture is unified to make it easy to add different compression methods.
+NNCF is designed to work with models from the two most popular deep learning frameworks, [PyTorch](https://pytorch.org/) 
```

Comment:
I would remove "the two most popular deep learning frameworks," and simply say "from PyTorch and TensorFlow"

Reply:
Done